### PR TITLE
Jazmin dev

### PR DESCRIPTION
**Pull request by jazvences**
add buttons, add new file for about me page, change background, and add project id

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -5,15 +5,33 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <style>
```
Comment (cort-p) [inline portfolio/src/main/webapp/index.html:8]: We generally prefer to keep styles separate from the html, so best to move this to the css file. See the style guide for some more info here, https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -5,15 +5,33 @@
...
+    </style>
+    <link href="https://fonts.googleapis.com/css?family=Annie+Use+Your+Telescope|Nanum+Pen+Script&display=swap" rel="stylesheet">
   </head>
   <body>
+      <div class="nav">
+  <a class="active" href="#home">Home</a>
+  <a href="#news">News</a>
+  <a href="#contact">Contact</a>
+  <a href="about.html">About Me</a>
+  </div>
```
Comment (cort-p) [inline portfolio/src/main/webapp/index.html:21]: Minor style thing here, try to keep the indentation consistent and indent nested elements more than the parent. Our style guide recommends 2 spaces, https://google.github.io/styleguide/htmlcssguide.html#Indentation.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -7,3 +7,62 @@
...
+.linkedin {
+    width: 50px;
+    margin: 3px;
+}
+
+.gmail {
+    height: 50px;
+    margin: 3px;
+}
+
```
Comment (cort-p) [inline portfolio/src/main/webapp/style.css:29]: Minor nit: use 2 spaces of indentation above (as you do below).

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -5,15 +5,33 @@
...
     <div id="content">
+    <div style="background-color:white;background-image:url(//www.html.am/images/backgrounds/background-image-2.gif);background-repeat:repeat;"></div>
       <h1>My Portfolio</h1>
       <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <br>
+      <p>This is my domain!~ Welcome :) In this webpage, I will be explaining a bit more about myself. I hope you enjoy your stay!</p>
       <div id="greeting-container"></div>
```
Comment (cort-p) [inline portfolio/src/main/webapp/index.html:28]: Is this greeting-container needed? I don't think it is really doing anything right?

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -7,3 +7,62 @@
...
+img {
+
```
Comment (cort-p) [inline portfolio/src/main/webapp/style.css:31]: Super minor: remove this blank line

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -7,3 +7,62 @@
...
+.nav a.active {
+  background-color: #80ffbf;
+  color: grey;
+}
+
+h1,h3,h4 {
+    text-decoration: underline;
+    font-size: xx-larger;
+    font-family: 'Nanum Pen Script', cursive;
+}
```
Comment (cort-p) [inline portfolio/src/main/webapp/style.css:63]: Consider combining the font-size and font-family properties into the single font property, similar to here in the style guide, https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties.

File: portfolio/src/main/webapp/style.css [hunk 7]
```diff
@@ -7,3 +7,62 @@
...
+  background-color: #ffcccc;
+  color: grey;
+}
+
+.nav a.active {
+  background-color: #80ffbf;
+  color: grey;
+}
+
+h1,h3,h4 {
```
Comment (cort-p) [inline portfolio/src/main/webapp/style.css:59]: Another minor point, we generally put separate selectors on separate lines, https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation.

**Review by cort-p — CHANGES_REQUESTED** [review]
Looking good! Mostly just some stylistic things to clean up.

File: portfolio/src/main/webapp/index.html (hunk 4 above)
Comment (jazvences) [reply portfolio/src/main/webapp/index.html:28]: It was there by default! I forgot to remove it, haha.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (jazvences) [reply portfolio/src/main/webapp/index.html:21]: Got it! I haven't gotten around to fixing the small details yet, but i will keep a look out for that.

File: portfolio/src/main/webapp/style.css (hunk 6 above)
Comment (jazvences) [reply portfolio/src/main/webapp/style.css:63]: Okay!

**Review by jazvences — COMMENTED** [review]
Resolving suggestions within next few pull requests.

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 8]
```diff
@@ -13,6 +13,6 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1>Hello Jazmin!</h1>");
```
Comment (cort-p) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java:16]: Ideally include this change in a separate PR.

**Review by cort-p — APPROVED** [review]

**Pull request merged**